dtreeviz/trees.py

```diff
@@ -176,8 +177,12 @@ def ctree_leaf_distributions(self,
             Plot labels font name
         :param grid: bool
             True if we want to display the grid lines on the visualization
-        :param figsize: optional (width, height) in inches for the entire plot
-            :param ax: optional matplotlib "axes" to draw into
+        :param label_all_leaves: bool
```

Comment:
I wonder if it's better to call it `show_leaf_labels`. I think we use *show* a lot.

Reply:
@parrt how would you like to procede from here? Should I change the param name and we'll merge this PR, or do we want to table this and fix it in a more elegant way?

Reply:
@tlapusan when you have time, maybe you have thoughts on this as you built those functions/plots?

Reply:
> I wonder if it's better to call it `show_leaf_labels`. I think we use _show_ a lot.

Agree with this. We have arguments for other methods which start with show_...

Reply:
I changed it `label_all_leaves` -> `show_leaf_labels`